packages/bpk-component-nudger/src/bpk-nudger.scss

```diff
@@ -21,6 +21,11 @@
 .bpk-nudger {
   &__button {
     position: relative;
+    @include prop(color, secondary-color);
+
+    &[disabled] {
```

Comment:
Why did you change this to not be based on class name?

Reply:
The classname was being set to the icon instead of the button itself. And now we set the fill to currentColor which is the button font color.